apps/accelerate/chatllama/README.md

```diff
@@ -48,6 +48,7 @@ First, let’s get the artifacts for running ChatLLaMA. The artifacts contain:
 
 - `config.yaml`: config file for model and data set. This allows you to 1) select the model you prefer (LLaMA, OPT, BLOOM, etc) 2) change all the hyperparameters of the training process;
 - `ds_config.json`: config file to define DeepSpeed training parameters;
+- `peft_config.yaml`: config file to define PEFT parameters;

```

Comment:
It would make sense to add a sentence explaining how peft can be used. Something like 
```suggestion
- `peft_config.yaml`: config file to define PEFT parameters; PEFT is used for efficient training with HuggingFace models. It can be used for setting the LoRA parameters as rank and precision.

```